keras_nlp/src/models/backbone.py

```diff
@@ -198,7 +199,11 @@ class like `keras_nlp.models.Backbone.from_preset()`, or from
         )
         ```
         """
-        validate_metadata(preset)
+        format = check_format(preset)
+
+        if format == "huggingface":
```

Comment:
I know this is a comment on my own code here. But we should probably rename this from `"huggingface"` to `"safetensors"` or `"transformers"` everywhere.